postbuild_fixups.sh

```diff
@@ -26,4 +26,13 @@ do
 done
 
 # Artifacts.toml
-# sed -i "s/[1-999]\.[0-999]\.[0-999]\|{version}/${version}/" Artifacts.toml
+julia -e 'using TOML, SHA, Tar, Inflate; \
```

Comment:
Hmm, is `Inflate` part of the stdlib? Or do we need to install it?

Reply:
first of all, don't google `julia Inflate` it was a bad idea and I regret it. 

No it's not `stdlib` 😕, we need to install it.

Reply:
Suggestion

```diff
--- a/postbuild_fixups.sh
+++ b/postbuild_fixups.sh
@@ -26,7 +26,8 @@ do
 done

 # Artifacts.toml
-julia -e 'using TOML, SHA, Tar, Inflate; \
+julia -e 'using Pkg; Pkg.activate(; temp=true); Pkg.add("Inflate"); \
+    using TOML, SHA, Tar, Inflate; \
     version = TOML.tryparsefile("Project.toml")["version"]; \
     filename = "deps/deps.tar.gz"; \
     res_key = "dash_textarea_autocomplete_resources"; \
```

what do think? This way we don't pollute the Project.toml with `Inflate`.

Reply:
Great! I'll add it